draft-ietf-quic-transport.md

```diff
@@ -2801,6 +2877,9 @@ is only sending ACK frames will only receive acknowledgements for its packets
 if the sender includes them in packets with non-ACK frames.  A sender SHOULD
 bundle ACK frames with other frames when possible.
 
+Implementations must be aware of the packet number space of the packet being
+transmitted and only include the ACK frames of the packets from that space.
```

Comment:
"ACK frames of the packets" is awkward phrasing.  "...and only acknowledge packets from that space," maybe?

Reply:
Done.